docs/static/templates/document-planning-template.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Doc planning template
+menuTitle: Doc Planning
+description: Use this template when you write a documentation plan.
+aliases:
+  - /docs/writers-toolkit/latest/templates/document-planning-template
+weight: 100
+keywords:
+  - doc planning
+  - documentation plan
+  - writing new documents
+---
+<!-- Refer to [Topic front matter]({{< relref "../../front-matter/" >}}) for more information about how to populate front matter. -->
```

Comment:
Is this comment left over from the template?  Does it need to be here?

Reply:
Good catch. Deleting.